cfg/config.json

```diff
@@ -380,5 +381,26 @@
         "scope": "email"
       }
     }
+  },
+  "importJobs": {
```

Comment:
The job is added in the DataImporter - it should be added to this repo - https://github.com/restorecommerce/data/tree/master/demo-shop/data

Reply:
Please rename `unactivatedAccountExpiry` --> `inactivatedAccountExpiry`